.gitmodules

```
@@ -88,3 +88,7 @@
 	path = 3rdparty/rtmidi/rtmidi
 	url = ../../thestk/rtmidi
 	ignore = dirty
+[submodule "3rdparty/zstd/zstd"]
+	path = 3rdparty/zstd/zstd
+	url = https://github.com/facebook/zstd
```

I think this must also be
```suggestion
	url = ../../facebook/zstd
```